pallets/parachain-staking/src/lib.rs

```diff
@@ -2859,13 +2900,22 @@ pub mod pallet {
 				DispatchClass::Mandatory,
 			);
 
-			let collators = Pallet::<T>::selected_candidates().to_vec();
-			if collators.is_empty() {
+			let selected_candidates = Pallet::<T>::selected_candidates().to_vec();
```

Comment:
Is that possible that we don't do the delayed distribution while the session switching? I'm not sure whether the computation is too high because, during the session switching, we have to do the snapshot, too

Reply:
Here we're only taking snapshot, no payouts are happening